apps/dcos_dns/src/dcos_dns_handler.erl

```diff
@@ -151,21 +146,18 @@ resolve_loop(Begin, [], Fun, FailMsg, Zone) ->
 resolve_loop(Begin, Workers, Fun, FailMsg, Zone) ->
     receive
         {reply, Pid, Response} ->
-            {value, {Upstream, Pid, MonRef}, Workers0} =
+            {value, {_Upstream, Pid, MonRef}, Workers0} =
```

Comment:
Oh, I overlooked this one. It seems like nobody uses the first element of this tuple. If so, could you please remove it?